Objects/Objects/ObjectsKit.cs

```diff
@@ -84,7 +84,7 @@ public ISpeckleConverter LoadConverter(string app)
         }
         else
         {
-          throw new SpeckleException($"Converter for {app} was not found in kit {basePath}");
+          throw new SpeckleException($"Converter for {app} was not found in kit {basePath}", level: Sentry.Protocol.SentryLevel.Warning);
```

Comment:
Should log be true?

Reply:
yep agreed on this - if we end up here something seriously borked in the installation process...